moz_telemetry/sandboxes/heka/input/moz_telemetry_heka_kafka.lua

```diff
@@ -0,0 +1,73 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This seems like the kafka/sandboxes/heka/input/heka_kafka.lua should be generalized to accepted a decoder and if nil uses the existing behavior (pass through injection).  To keep the terminology the same as Heka it should be a 'decoder' module.